stride/assetlist.json

```diff
@@ -403,6 +403,43 @@
         }
       ]
     },
+    {
+      "description": "Stride's liquid staked SAGA",
```

Comment:
Can you please relocate this to the bottom of the list of assets?

Reply:
done